Gemfile

```diff
@@ -27,7 +27,9 @@ gem 'jbuilder', '~> 2.7'
 # Reduces boot times through caching; required in config/boot.rb
 gem 'bootsnap', '>= 1.4.2', require: false
 gem 'bootstrap', '~> 4.4.1'
+gem 'haml-rails', '>= 2.0', group: :development
```

Comment:
It looks like you're including changes not related to the purpose of this PR, which is merely adding the PostgreSQL configuration